### ISSUE TITLE
Advanced Search - "Related To" incorrectly taking name only and not ID

### ISSUE THREAD

**Issue by jellydonuts** [salesagility/SuiteCRM#1233]
It looks like when you do an advanced search and select the related to: field as account and directly specify ONE account, the search still runs under the name of the account, not the unique ID of the related account and hence, if you have multiple accounts with a similar naming convention, your specific advanced search will not run as intended. 

See screenshot for example

![adv_search_too_wide](https://cloud.githubusercontent.com/assets/1541720/14475662/fad824e0-00b7-11e6-8518-047c9842f340.jpg)

the line from my debug log is as follows:
`Tue Apr 12 14:09:07 2016 [27528][1][INFO] List View Where Clause: (tasks.parent_type like 'Accounts%' ) AND ( parent_Accounts_accounts.name like 'RBC%')`

this should instead reference the parent_Accounts_accounts.id or something more unique

**Comment by jellydonuts** [comment]
I'm not really sure where to look for this one. I would guess and say it's some underlying Sugar search logic (that's used throughout all modules) so I don't know which core files i can look into to 'hack up' a solution... some pointers would be greatly appreciated if no fix is foreseen.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Advanced Search - "Related To" incorrectly taking name only and not ID](https://trello.com/c/xLLj56FZ/106-advanced-search-related-to-incorrectly-taking-name-only-and-not-id)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
